### PR TITLE
Enable get_current_url and get_title wdspec tests

### PR DESCRIPTION
**Pull request by georgeroman**
---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors

- [X] There are tests for these changes

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23548)

### CODE REVIEW

File: components/net/filemanager_thread.rs [hunk 1]
```diff
@@ -779,7 +779,10 @@ pub fn fetch_file_in_chunks(
                     return;
                 }
                 let length = {
-                    let buffer = reader.fill_buf().unwrap().to_vec();
+                    let buffer = match reader.fill_buf() {
+                        Ok(buf) => buf.to_vec(),
+                        Err(_) => vec![],
```
Comment (Manishearth) [inline components/net/filemanager_thread.rs]: Shouldn't we have a way of bubbling this error state out?

Comment (georgeroman) [reply components/net/filemanager_thread.rs]: With this change, an `Unsupported content type: application/octet-stream` error or something similar is displayed as text in the window.

Comment (Manishearth) [reply components/net/filemanager_thread.rs]: How?

Comment (georgeroman) [reply components/net/filemanager_thread.rs]: This is what I get if I try opening a directory:

servo

**Pull request closed without merging**